decidim-comments/spec/forms/comment_form_spec.rb

```diff
@@ -103,6 +105,31 @@ module Comments
           end
         end
       end
+
+      shared_examples "allows commenting" do
+        it "allows commenting" do
+          expect(subject.send(:commentable_can_have_comments)).to be_nil
+          expect(subject).to be_valid
+        end
+      end
+
+      describe "#commentable_can_have_comments" do
+        before do
+          allow(subject).to receive(:current_user).and_return(current_user)
```

Comment:
instead of this, just add a new entry in the context definition at the begging of the file:
```
.with_context(
          current_organization: organization,
          current_component: component,
          current_user: user
        )
```

Reply:
Done